zuul.d/projects.yaml

```diff
@@ -110,6 +110,14 @@
             branches: master
         - docker-machine-0.15.0-functional-devstack-rocky:
             branches: master
+
+- project:
+    name: kubernetes-sig/cluster-api-provider-openstack
```

Comment:
s/kubernetes-sig/kubernetes-sigs/

This wrong project name cause CI failing, and I have added `kubernetes-sigs/cluster-api-provider-openstack` into zuul main.yaml config file, update your patch and input `recheck` in PR comments, I think everything should be ok.

Reply:
ooops! Thanks a lot!